cli/src/actions/description.ts

```diff
@@ -0,0 +1,24 @@
+import type { Context } from "@/context";
```

Comment:
This is the new file I added to ask for a description of the app. This is what is ultimately used for code gen